resources/lib/kodiwrapper.py

```diff
@@ -379,6 +372,13 @@ def get_global_setting(self, setting):
         result = self.jsonrpc(method='Settings.GetSettingValue', params=dict(setting=setting))
         return result.get('result', {}).get('value')
 
+    def set_global_setting(self, setting, value):
```

Comment:
See my notes in Slack for my perspective on logging. But a good rule for add-ons would be that they do not influence Kodi settings directly, as it may be confusing since it could also influence how other add-ons behave. (And we don't do that either for the other global settings we support)

On the other hand, it would be nice to jump directly to Kodi settings-tabs and highlight a setting (to then return back to the add-on).